specification/mongocluster/DocumentDB.MongoCluster.Management/examples/2024-03-01-preview/MongoClusters_PatchDiskSize.json

```diff
@@ -1,60 +0,0 @@
-{
-  "operationId": "MongoClusters_Update",
```

Comment:
Why do you remove the example?
If there are errors in the example, you can fix it or skip it in codegen